python/ray/serve/examples/tutorials/sklearn+composition.ipynb

```diff
@@ -0,0 +1,527 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Serve Tutorial: scikit-learn and Model Compositions\n",
+    "\n",
+    "In this tutorial, we will show you:\n",
+    "- how to deploy scikit-learn models with Ray Serve\n",
+    "- how to compose multiple models together using RayServeHandle\n",
+    "\n",
+    "[scikit-learn](https://scikit-learn.org/) is a popular library with easy to use classification and regression models. In this tutorial, we will train several models to tackle a simple classification task with the iris dataset and compose them together."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install dependencies\n",
+    "!pip install scikit-learn numpy ray"
```

Comment:
Is ! intentional?

Reply:
The `!` tells jupyter to run a bash command